scripts/release.ts

```
@@ -0,0 +1,62 @@
+import { promises as fs } from 'fs';
+import execa from 'execa';
+import chalk from 'chalk';
+import { prependChangelog } from 'changelog-prepend/lib/prependChangelog';
+
+process.env.GITHUB_AUTH = process.env.GITHUB_AUTH || process.env.github_token;
+
```

I don't know if this is handled in the library or not, but I would throw an error in case this wasn't supplied.

---

lerna-changelog exit code is 0, but the output is handled by the library, and it throws an error:
https://github.com/wix-a/changelog-prepend/blob/master/packages/changelog-prepend/src/generateNewChangelog.ts#L15

---

done https://github.com/wix/yoshi/pull/2726/files#diff-1d7f42573d5c3507c48e94f5ed470cdfR16